subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -496,7 +496,10 @@ struct event_done_extra {
 #endif /* CONFIG_BT_CTLR_JIT_SCHEDULING */
 	union {
 		struct {
-			uint16_t trx_cnt;
+			union {
+				uint32_t trx_performed_mask;
```

Comment:
Do you need 32-bits?

```suggestion
				uint16_t trx_performed_mask;
```

Reply:
We actually DO need 32 bits as there can be up to 31 CISs - this is a mask with bits set per CIS with activity